### PR TITLE
gdu: 2.0.0 -> 2.1.0

### PR DESCRIPTION
**Pull request by zowoq**
https://github.com/dundee/gdu/releases/tag/v2.1.0

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 108458` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>gdu</li>
  </ul>
</details>

File: pkgs/tools/system/gdu/default.nix [hunk 1]
```diff
@@ -5,17 +5,19 @@
...
     owner = "dundee";
     repo = pname;
     rev = "v${version}";
-    sha256 = "0239ppiilr8d43z9v27c9h7b5xkj2n9aacpf5a0h3rz4j0dkcwf7";
+    sha256 = "0gk36z8xzj7blwzs080fqsz76hn56c89xcsyil6n5cfkbyf85c6i";
   };
 
   vendorSha256 = "1jqbsda9bch3awdq816w4jybv7wz9mfflmvs5y2wsa2qnhn9nbyp";
 
+  buildFlagsArray = [ "-ldflags=-s -w -X main.AppVersion=${version}" ];
```
Comment (fabaff) [inline pkgs/tools/system/gdu/default.nix:19]: Thanks for fixing it. Looks like that upstream is always using [`development`](https://github.com/dundee/gdu/blob/v2.0.0/main.go#L16) and not updating it.

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 108458` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>gdu</li>
  </ul>
</details>

**Review by fabaff — APPROVED** [review]
- [x] package build (see https://github.com/NixOS/nixpkgs/pull/108458#issuecomment-754486418)
- [x] executables tested on x86_64

```bash
$ ./gdu -h
Usage of ./gdu:
  -ignore-dir string
        Absolute paths to ignore (separated by comma) (default "/proc,/dev,/sys,/run")
  -log-file string
        Path to a logfile (default "/dev/null")
  -no-color
        Do not use colorized output
  -v    Prints version
```

- [x] all depending packages build

Looks good to me

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 108458` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>gdu</li>
</ul>
</details>

**Pull request merged**